hotsos/core/plugins/openvswitch/ovs.py

```diff
@@ -200,6 +207,70 @@ def offload_enabled(self):
         return False
 
 
+class OVSBFD(OpenvSwitchBase):
+    # pylint: disable=C0301
```

Comment:
Instead of disabling the long line warning we can allow unbreakable long lines (such as URLs) with the following pylint option:

https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#ignore-long-lines

Reply:
ill try it out